### ISSUE TITLE
Log everything to a file

### ISSUE THREAD

**Issue by jacobilsoe** [rtcTo/rtc2git#62]
It would be nice if everything was logged to a file - not only the accept messages.

**Comment by WtfJoke** [comment]
Yeah I think thats a good idea. I struggled a few times when I lost some logs....
I think its possible by extending shell#execute.

**Comment by WtfJoke** [comment]
Its not as easy as it seems. If a file is provided I need to log to two files (the normal log file and the provided one)... Not sure if I can accomplish this. It might be possible to work out with some kind of decorator pattern.

**Comment by ohumbel** [comment]
what about 
`python3 migration.py > migration.log  2>&1`
and then 
`tail -f migration.log`

**Comment by WtfJoke** [comment]
True on linux. Im not sure if there is a similar solution for windows.
It might be possible on windows with ```python migration.py > migration.log``` and an editor of your choice which supports autoreloading...
